custom_components/smartthinq_sensors/wideq/devices/ac.py

```diff
@@ -866,6 +869,24 @@ async def get_filter_state_v2(self):
             self._filter_status_supported = False
             return None
 
+    async def set_reservation_sleep_time(self, value: float):
```

Comment:
Why you use a float? As I understand input value should represent number of minutes and can be of type `int`

Reply:
I wasn't sure how to handle this. LG's API returns 'reservation sleep time' as `float` and I tried to mirror LG's API's name conventions and signatures. How ever I haven't seen any value with decimal component (≠ .0).

May be I can "ceil" anything coming from LG's API and use `int`s for anything else

@ollo69 what do you think?

Reply:
If LG API need a float in the call, you could use `int` as input parameter for the function and than just cast it to `float` in the API call. If you start from a `int` you don't need `ceil`